### DISCUSSION TITLE
upgrade http-server to latest version to avoid security issue

### DISCUSSION THREAD

**Discussion by arthurgeek** [nrwl/nx#7916] [Feature Requests]
Hi!

The [http-server](https://www.npmjs.com/package/http-server) version used by `web` depends on a very old version of [ecstatic](https://www.npmjs.com/package/ecstatic). This package is no longer maintained and newer versions of `http-server` have already removed that dependency. That version also contains a [known security problem](https://github.com/advisories/GHSA-jc84-3g44-wf2q).

I tried doing it myself and open a PR, but I'm very confused about the way dependencies are managed inside the NX monorepo since the `http-server` dependency is listed twice, one in the main `package.json` and in the web package own `package.json` file. The contributing guidelines have no mention of this and I couldn't find elsewhere.

**Comment by puku0x** [comment]
It looks like it was fixed on [v13.4.2](https://github.com/nrwl/nx/releases/tag/13.4.2)

https://github.com/nrwl/nx/pull/8334
